test/test_response.py

```diff
@@ -49,6 +50,18 @@ def sock():
     s.close()
 
 
+class TestContentDecoder:
+    decoder = ContentDecoder()
+
+    def test_decompress_error(self):
+        with pytest.raises(NotImplementedError):
```

Comment:
What are your thoughts on adding `raise NotImplementedError` to the `.coveragerc` skip list? Then we don't need mostly no-op tests like this to cover those cases